modules/google-analytics/readme.txt

```diff
@@ -0,0 +1,133 @@
+=== WP Google Analytics ===
```

Comment:
Since it appears we're using non-a8c GPL code, we need to be sure to properly acknowledge that.